### PR TITLE
CORE-2263: Send all traces to zipkin

### PR DESCRIPTION
**Pull request by benwaffle**
The purpose of this PR is to send all traces as instrumented by Kamon to Flow's zipkin instance. This will allow us to view all intra-service API calls for a given external API call, as identified by the `trace-id` header.

Kamon collects traces of all HTTP calls (incoming and outgoing), actor messages, and database queries. No HTTP headers, bodies, or DB query parameters are collected. https://kamon.io/docs/latest/instrumentation/

Flow's zipkin instance is at https://zipkin.flo.pub

### CODE REVIEW

**Review by nemo83 — APPROVED** [review]
Looking forward to see this in action.

**Review by jlogeart — APPROVED** [review]
🚀

**Review by paololim — APPROVED** [review]
this is fine, just make sure that during the deploy, we see that it actually doesn't break anything

File: conf/application.production.conf [hunk 1]
```diff
@@ -1,3 +1,8 @@
 include "base.conf"
 include "/flow-metrics.conf"
 
+kamon.zipkin = {
+  username = "submit"
+  password = ${?ZIPKIN_PASSWORD}
```
Comment (paololim) [inline conf/application.production.conf:6]: Is this required?

Comment (benwaffle) [reply conf/application.production.conf:6]: Yes, because proxy is outside our k8s cluster

**Pull request merged**